master/reference/networking-calico/implementation-notes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Implementation notes
```

Comment:
Discarding as you say, and will leave in the networking-calico repo.  Ditto for the DHCP agent page.